frontend/src/components/VHomeGallery/VHomeGallery.vue

```diff
@@ -55,7 +55,7 @@ import useResizeObserver from "~/composables/use-resize-observer"
 
 import VLink from "~/components/VLink.vue"
 
-import imageInfo from "~/assets/homepage_images/image_info.json"
+import imageInfo from "~/assets/ homepage_images.json"
```

Comment:
```suggestion
import imageInfo from "~/assets/homepage_images.json"
```
There is an extra space here. The file name also has a space, could you please remove it?

Reply:
Pushed the changes, aplogies for the silly mistake.

Reply:
No problem, it happens to everyone :)